src/lib/codemirror-editor.ts

```diff
@@ -58,6 +58,11 @@ export class CodeMirrorEditorElement extends LitElement {
     .cm-wrap {
       height: 100%;
     }
+
+    /* Hide the caret when editor is not focused. */
+    .cm-wrap:not(.cm-focused) .cm-cursor {
+      display: none;
```

Comment:
Does CodeMirror have a programmatic way of hiding the cursor? Wondering in case CM is using a timer to make blinking cursors and whether we want to turn off the timers. It could be using CSS too, just unsure.

Reply:
I looked into this before, couldn't find anything. It uses a CSS animation though.